### PR TITLE
chore: Add LDS UIAPI Adapters to base.js

### PR DESCRIPTION
**Pull request by dme722**
Adding missing LDS adapters to our lint rules.

### CODE REVIEW

File: base.js [hunk 1]
```diff
@@ -263,6 +271,62 @@ const WIRE_ADAPTERS_WITH_RESTRICTED_USE = [
...
+    {
+        module: 'lightning/uiRecordActionsApi',
+        identifier: 'getRelatedListActions',
+    },
+    {
+        module: 'lightning/uiRecordActionsApi',
+        identifier: 'getRelatedListsActions',
+    },
+    {
+        module: 'lightning/uiLayoutApi',
```
Comment (kevinv11n) [inline base.js:315]: This is not accessible by customers

**Review by kevinv11n — CHANGES_REQUESTED** [review]
@dme722 only customer-accessible wire adapters should appear in this repo. This PR contains files that aren't customer accessible.

**Comment by dme722** [comment]
Thanks @kevinv11n, I removed the ones that aren't customer facing

File: base.js [hunk 2]
```diff
@@ -263,6 +271,38 @@ const WIRE_ADAPTERS_WITH_RESTRICTED_USE = [
         module: 'lightning/uiRecordApi',
         identifier: 'getRecordUi',
     },
+    {
+        module: 'lightning/uiRelatedListApi',
+        identifier: 'getRelatedListsCount',
+    },
```
Comment (kevinv11n) [inline base.js:277]: I don't see this export

File: base.js [hunk 3]
```diff
@@ -263,6 +271,38 @@ const WIRE_ADAPTERS_WITH_RESTRICTED_USE = [
...
+    {
```
Comment (kevinv11n) [inline base.js:278]: Can you alphabetize this file to match the lightning modules so it's easier to track?

**Review by kevinv11n — CHANGES_REQUESTED** [review]
Still some fixes needed.

A script to generate this file would be great to avoid these issues.

File: base.js (hunk 2 above)
Comment (dme722) [reply base.js:277]: I was going based on the fact that the module `uiRelatedListApi` is exposed per [this js-meta.xml file](https://github.com/salesforce/lightning-components/blob/master/ui-lightning-components/src/main/modules/interop/uiRelatedListApi/uiRelatedListApi.js-meta.xml). Is that an incorrect way to approach this?

Comment (dme722) [reply base.js]: Should be fixed now!

**Review by nolanlawson — APPROVED** [review]

**Pull request merged**